docs/user/managing-projects-feeds.md

```diff
@@ -52,10 +52,37 @@ Feed Versions are created from the main Feed Source profile page. There are thre
 
 3. **Import From the GTFS Editor**: Select "From snapshot" from the `+ Create new version` dropdown. The list of snapshots should now be visible showing any available snapshots of the feed in the Editor. Select the desired snapshot by clicking the "Publish" button to publish the snapshot as a new version.
 
+4. **Service Period Merge** (certain Data Tools configurations only): If a Feed Source has two or more Feed Versions, a new Feed Version can be created by merging two versions representing a transit agency's service over different time periods. While viewing a particular Feed Version, click **Merge with version** (underneath the map view) to select which past version you would like to merge with.
+
+4. **Regional Merge**: For Projects that contain multiple Feed Sources across a region, it can be useful to merge multiple transit agencies together into a combined GTFS feed for the entire region. While viewing a Project's list of Feed Sources, click **Actions > Merge all** to produce a combined GTFS file for all Feed Sources.
```

Comment:
"It can be useful to merge **feeds from** multiple transit agencies...".

"Actions > Merge all" should be outlined with backquote (\`) to be consistent with the rest of the document.

Reply:
Bullet number should be 5 (it renders correctly nonetheless).